docs/csharp/language-reference/keywords/try-catch.md

```diff
@@ -87,9 +87,19 @@ catch (InvalidCastException e)
     {  
         // Take some action.  
     }  
- }  
+}  
 ```  
-  
+
+> [!NOTE]
+> It is also possible to use a predicate expression to get a similar result in a often cleaner fashion. The following example has a similar behavior for callers than the precedent example, the function throws the `InvalidCastException` back to the caller when `e.Data` is null.
```

Comment:
Instead of "predicate expression", I'd prefer the term "exception filter".

Reply:
Some additional suggested changes, @sylveon: 
- a often -->an often
- than the precedent example, --> as the previous example.
- the function --> The function
- null --> \`null\`

Reply:
I was following the name used a bit before in the same document.

```
Using catch arguments is one way to filter for the exceptions you
want to handle. You can also use a predicate expression that further
examines the exception to decide whether to handle it. If the
predicate expression returns false, then the search for a handler continues.
```

Should I change that too while I'm at it @BillWagner?

@rpetrusha will do.

Reply:
thanks @sylveon Yes, let's change it in both locations

Reply:
Done and done.